modules/navier_stokes/doc/content/source/fvbcs/INSFVMomentumAdvectionOutflowBC.md

```diff
@@ -0,0 +1,13 @@
+# INSFVMomentumAdvectionOutflowBC
+
+This object implements the $\rho\left(\vec u \cdot\nabla\right)\vec u$ component
+term of the
+incompressible Navier Stokes momentum equation along a domain boundary. It
+simultaneously requires that the normal gradient of each velocity component at
+the boundary be zero.
```

Comment:
It does this later point by tagging the surface as fully developed flow (fdf) right

Reply:
yes